Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -426,6 +426,7 @@ granularity.years=Jahrg\u00E4nge
 guessImages=gesch\u00E4tzte Seitenanzahl
 help=Hilfe
 hideCorrectionTasks=Entferne Korrekturschritte
+HierarchyMigrationTask=Migriere serielle Publikationen
```

Comment:
please use lowecase

Reply:
I have now changed this consistently so that all long-running task classes now use their class name for the name, starting with a lowercase letter.